tmail-backend/tmail-third-party/openpaas/pom.xml

```diff
@@ -67,6 +72,11 @@
             <artifactId>mockserver-netty</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.linagora.tmail</groupId>
+            <artifactId>tmail-guice-distributed</artifactId>
```

Comment:
Yes this dependecy here is evil and should be removed at all costs.

Move the test in `distributed` maven module?

(Good catch @Arsnael )